plugins/basicstyles/plugin.js

```diff
@@ -101,6 +101,41 @@ CKEDITOR.plugins.add( 'basicstyles', {
 			[ CKEDITOR.CTRL + 73 /*I*/, 'italic' ],
 			[ CKEDITOR.CTRL + 85 /*U*/, 'underline' ]
 		] );
+	},
+
+	afterInit: function( editor ) {
+		var subscriptCommand = editor.getCommand( 'subscript' ),
+			superscriptCommand = editor.getCommand( 'superscript' ),
+			disallowAddSubSup = editor.config.coreStyles_disallowSubscriptSuperscript;
```

Comment:
We can't go with that name for the configuration option. It indicates that the option disables sub and sup script features, not make then exclusive.